src/main/java/com/mixpanel/android/mpmetrics/MPConfig.java

```diff
@@ -56,6 +56,7 @@ public static MPConfig getInstance(Context context) {
         mBulkUploadLimit = metaData.getInt("com.mixpanel.android.MPConfig.BulkUploadLimit", 40); // 40 records default
         mFlushInterval = metaData.getInt("com.mixpanel.android.MPConfig.FlushInterval", 60 * 1000); // one minute default
         mDataExpiration = metaData.getInt("com.mixpanel.android.MPConfig.DataExpiration",  1000 * 60 * 60 * 24 * 5); // 5 days default
+        mDBMaxSize = metaData.getInt("com.mixpanel.android.MPConfig.DBMaxSize",  20 * 1024 * 1024); // 20 Mb
```

Comment:
This name isn't really the Max size (we'll allow it to take up half the available storage) - I'm not sure what the name should be, maybe "MinimumDatabaseLimit"? Then if the user sets it to zero, we will demand that there is at least half the available space left.